### PR TITLE
Link up screening component on specific movie page

### PR DESCRIPTION
**Pull request by christerresin**
Screenings displayed on movie page is now clickable links to booking page. Linked to  /booking with querystring

ex. _/booking?screeningId=2202205012200_

ScreeningId = [movieId][date][time] = [2][20220501][2200] = 2202205012200

Database for movies updated with above pattern for screenings

### CODE REVIEW

**Comment by christerresin** [comment]
Updated .env.local and dbConnect.js to use MONGODB_MOVIES_URI. Noticed David is using the same URI "placeholder" inside .env

**Comment by stratosfloor** [comment]
> Screenings displayed on movie page is now clickable links to booking page. Linked to /booking with querystring
> 
> ex. _/booking?screeningId=2202205012200_
> 
> ScreeningId = [movieId][date][time] = [2][20220501][2200] = 2202205012200
> 
> Database for movies updated with above pattern for screenings

Smart for easily creating unique screeningIDs.

**Comment by stratosfloor** [comment]
Not much to say, clean code. Maybe the backBtn should be in on MoviePage instead of inside reviews-component.

**Review by stratosfloor — APPROVED** [review]

**Comment by christerresin** [comment]
> Not much to say, clean code. Maybe the backBtn should be in on MoviePage instead of inside reviews-component.

Good input, the placement of the button inside the reviews component is "wrong". My first thought was having it as close to the "back" as possible. Ill go ahead and move it before merge

File: lib/dbConnect.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import mongoose from 'mongoose';
 
-const MONGODB_URI = process.env.MONGODB_URI;
+const MONGODB_URI = process.env.MONGODB_MOVIES_URI;
 
 if (!MONGODB_URI) {
```
Comment (Erm0es) [inline lib/dbConnect.js:5]: I like this error handeling! Looks clean and easily understood.

**Review by Erm0es — APPROVED** [review]
I have no remarks on this code, It looks good!

**Comment by stratosfloor** [comment]
> Screenings displayed on movie page is now clickable links to booking page. Linked to /booking with querystring
> 
> ex. _/booking?screeningId=2202205012200_
> 
> ScreeningId = [movieId][date][time] = [2][20220501][2200] = 2202205012200
> 
> Database for movies updated with above pattern for screenings

I have added all screenings from the movie-Collection according to this in screening-Collection. Bookingpage with query strings works on my branch, so it should work when everything is merged if linked properly.

**Review by stratosfloor — APPROVED** [review]

**Pull request merged**